adr/ADR-8.md

```diff
@@ -145,15 +150,21 @@ type KV interface {
 	// Put saves a value into a key
 	Put(key string, val []byte, opts ...PutOption) (seq uint64, err error)
 
-	// Delete purges the subject
+	// Create is a variant of Put that only succeeds when the key does not exist of last historic event is a delete or purge
+	Create(key string, val []byte) (seq uint64, err error)
+
+	// Update is a variant of Put that only succeeds when the most recent operation on a key has the expected sequence
```

Comment:
Still prefer revision here and think consistency from here through client impl important.